### PR TITLE
docs: Contribution-guidelines

### PR DESCRIPTION
**Pull request by dakshsinghrathore**
This PR solves the issue #3 
Kindly review the PR and merge it under Hacktoberfest-accepted label !

### CODE REVIEW

File: CONTRIBUTION_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Contribution Guidelines
+
+We welcome your contributions to help improve this package. Your efforts are greatly appreciated in making this project better for everyone. To contribute, please follow these guidelines:
+
+1. **Fork the Repository**: Begin by forking the repository and then clone it to your local machine.
+
+    ```
+    git clone https://github.com/your-username/repository-name.git
```
Comment (devcer) [inline CONTRIBUTION_GUIDELINES.md]: can you update this url to proper url `https://github.com/user-focus/marvin-utils.git`

**Review by devcer — CHANGES_REQUESTED** [review]
need an update

**Review by devcer — APPROVED** [review]

**Comment by devcer** [comment]
Thank you for your contribution. Happy Hacktoberfest.

**Comment by dakshsinghrathore** [comment]
Kindly add the hacktoberfest-accepted label 😀🤝

**Pull request merged**